### ISSUE TITLE
Ability to toggle link previews

### ISSUE THREAD

**Issue by remainstheday** [UdaraJay/Pile#52]
Link previews are nice but can be distracting and ugly depending on the source's meta data. Some users might want a minimal distraction free experience.

![CleanShot 2024-01-02 at 10 43 18@2x](https://github.com/UdaraJay/Pile/assets/4063510/28dabf53-514a-416d-82c3-d53672514236)

**Comment by UdaraJay** [comment]
makes sense to have it as a per-entry option. could just store it in the entry's metadata as `hideLinkPreviews: bool`
